pkg/metrics/metrics.jsx

```diff
@@ -1667,33 +1676,68 @@
+                    const s = this.columns.find(itm => itm[1] == selection);
+                    this.setState({ selectedVisibility: { ...this.state.selectedVisibility, [s[0]]: !this.state.selectedVisibility[s[0]] } });
+                }}
+                onToggle={() => this.setState({ isOpenColumnVisibility: !this.state.isOpenColumnVisibility })}
+                placeholderText={_("Graph visibility")}
+                selections={selections}>
+                {columnVisibilityMenuItems}
+            </Select>
+        );
+
         return (
             <div className="metrics">
                 <div className="metrics-heading-sticky">
                     <section className="metrics-heading" style={{ "--has-swap": swapTotal ? "var(--column-size)" : "var(--half-column-size)" }}>
-                        <Select
-                            className="select-min metrics-label"
-                            aria-label={_("Jump to")}
-                            onToggle={this.handleToggle}
-                            onSelect={this.handleSelect}
-                            isOpen={this.state.isDatepickerOpened}
-                            selections={this.state.selectedDate}
-                            toggleId="date-picker-select-toggle"
-                        >
-                            {options}
-                        </Select>
+                        <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+                            <Select
+                                className="select-min metrics-label"
+                                aria-label={_("Jump to")}
+                                onToggle={this.handleToggle}
+                                onSelect={this.handleSelect}
+                                isOpen={this.state.isDatepickerOpened}
+                                selections={this.state.selectedDate}
+                                toggleId="date-picker-select-toggle"
+                            >
+                                {options}
+                            </Select>
+                            {graphsVisibilitySelector}
+                        </Flex>
                         <div className="metrics-graphs">
-                            <Label label={_("CPU")} items={[_("Usage"), _("Load")]} />
-                            <Label label={_("Memory")} items={[_("Usage"), ...swapTotal ? [_("Swap")] : []]} />
-                            <Label label={_("Disk I/O")} items={[_("Usage")]} />
-                            <Label label={_("Network")} items={[_("Usage")]} />
+                            {this.state.selectedVisibility.cpu && <Label label={_("CPU")} items={[_("Usage"), _("Load")]} />}
+                            {this.state.selectedVisibility.memory && <Label label={_("Memory")} items={[_("Usage"), ...swapTotal ? [_("Swap")] : []]} />}
+                            {this.state.selectedVisibility.disk && <Label label={_("Disk I/O")} items={[_("Usage")]} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18299-20230208-164058-692abce6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)